src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -18,6 +16,7 @@
 using ArchiveFileList = System.Collections.Generic.List<System.Tuple<string, string>>;
 using Mono.Cecil;
 using Xamarin.Android.Build.Utilities;
+using Xamarin.ZipSharp;
```

Comment:
API question/suggestion for another repo, but "Sharp" doesn't add much in terms of API. It should just be `Xamarin.Zip` or `Xamarin.Tools.Zip` or something.

(Not relevant for this PR, just a suggestion for the future.)